src/components/Widgets/OSMDataWidget/Messages.js

```diff
@@ -0,0 +1,16 @@
+import { defineMessages } from 'react-intl'
+
+/**
+ * Internationalized messages for use with OSMDataWidget
+ */
+export default defineMessages({
+  label: {
+    id: "Widgets.OSMDataWidget.label",
+    defaultMessage: "OSM Data",
```

Comment:
How to name this widget? "OSM Data", "OSM Element", "OSM Tags", "OSM Element Tags", other?

I will update file names to match the widget name once agreed on.

Reply:
Cool to see this contribution @homersimpsons!
I'd say "OSM Element Tags" would be most appropriate in this case. Open to other folks' opinons! @CollinBeczak ?

Reply:
I agree, that probably provides the best context for what the widget contains.

Reply:
Okay I renamed everything to "OSM Element Tags"